src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -443,18 +445,33 @@ function resolve(child, context) {
   return {child, context};
 }
 
+type Frame = {
+  domNamespace: string,
+  children: any,
+  childIndex: number,
+  context: Object,
+  footer: string,
+  debugElementStack: Array<any>,
```

Comment:
all the `Frame` annotations below only add `debugElementStack` in dev mode which causd a lot of errors in the logging utility functions. Just declaring it as always present (which it is in `dev`, and I think appropriately defended against in `prod`) solved a lot of the flow errors.